framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -9372,13 +9253,27 @@ void VulkanReplayConsumerBase::UpdateDescriptorSetInfoWithTemplate(
     }
 }
 
-VulkanDeviceAddressTracker& VulkanReplayConsumerBase::GetDeviceAddressTracker(VkDevice device)
+VulkanDeviceAddressTracker&
+VulkanReplayConsumerBase::GetDeviceAddressTracker(const decode::VulkanDeviceInfo* device_info)
 {
-    auto it = _device_address_trackers.find(device);
+    auto it = _device_address_trackers.find(device_info);
     if (it == _device_address_trackers.end())
     {
         auto [new_it, success] =
```

Comment:
Do we want to check success here? Report the error and abort?

Reply:
Cool, then at least an assert would make everyone happy.

Reply:
agreed, done :)